src/test/rpc/Tx_test.cpp

```diff
@@ -688,6 +730,29 @@ class Tx_test : public beast::unit_test::suite
                             result.second.transaction(),
                             txMeta,
                             tx);
+
+                        // Compare result to result from account_tx
+                        auto mentioned = tx->getMentionedAccounts();
+
+                        if (BEAST_EXPECT(mentioned.size()))
```

Comment:
My preference: reduce nesting by inverting these conditionals and turning them into early exits (`continue` in this situation). This one on line 737, but also the ones on lines 719, 722, and 744.